src/file.ts

```diff
@@ -1260,7 +1259,7 @@ class File extends ServiceObject<File> {
       const headers = {
         'Accept-Encoding': 'gzip',
         'Cache-Control': 'no-store',
-      } as Headers;
+      } as any;
```

Comment:
Maybe we should define an interface / type instead of using any here to avoid adding additional linter warnings.